### DISCUSSION TITLE
How to write in percentage Elast Alert CPU usage/Memory usage

### DISCUSSION THREAD

**Discussion by mdrajibkhan** [jertel/elastalert2#259] [Q&A] [answered]
Hi,

basically i want to know how to wrire a policy for cpu and memory usage in percentage and as well as notification will send on the slack. Thank you

**Comment by jertel** [comment, marked as answer]
Look into Metricbeat. If you have that running and logging metric data, including CPU/Memory data, into Elastic then you can write a rule to trigger if the value exceeds a threshold. Search this discussion forum for `metricbeat` and you'll find others who are doing this and their rules that you can use as a template.

**Reply by nsano-rururu** [reply]
https://github.com/jertel/elastalert2/blob/master/example_rules/example_single_metric_agg.yaml
https://github.com/manankalra/elastalert-tutorial/tree/master/rules
・elastalert_cpu_watch.yaml
・elastalert_memory_watch.yaml
